Bulls-and-cows/src/main/java/study/OperationMap.java

```diff
@@ -0,0 +1,30 @@
+package study;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum OperationMap {
+    PLUS("+", Integer::sum),
+    MINUS("-", (a, b) -> a - b),
+    DIVIDE("*", (a, b) -> a * b),
+    MULTIPLY("/", (a, b) -> a / b),
+    NONE("", (a, b) -> null);
+    private final String operation;
```

Comment:
```suggestion

    private final String operation;
```
열거형과 필드 사이 개행 부탁드릴게요!